src/target.c

```diff
@@ -126,10 +126,8 @@ stumpless_add_entry( struct stumpless_target *target,
     		return -1;
   	}
   	buffer = strbuilder_get_buffer( builder, &builder_length );
-	error_stream = stumpless_get_error_stream();
-  	if( fwrite( buffer, 1, builder_length, error_stream) != builder_length ){
-    	  return -1;
-	}
+	if( stumpless_log_to_error_stream( buffer, builder_length) == -1)
+      return -1;
```

Comment:
If perror write fails, the destination target is ignored.  Is this desired behavior?  After all, perror is just an option.

Reply:
Also, it returns without destroying the builder

Reply:
Falling through here is a valid option, though an error should still be raised (from within the log to error stream function) so that the write failure can be detected by callers.

Good catch on the builder not being destroyed!

Reply:
UPDATE: upon further inspection while working to fix this, I believe that @igorkole is correct on both fronts: the destination target shouldn't be ignored if the error stream writes fail. My apologies for not understanding the first time through. You'll be tagged as a coauthor in the commit that finally fixes this.